File: stock-app/src/entities/src/work.ts
import { Sheet, SectionSheet } from '../internal';
import { KeyNum } from './key-num';

export interface Work {
  id : string;
  projectName : string;
  master? : KeyNum;
  // match? : KeyNum;
  sections : SectionSheet;
}

export type WorkSheet = Sheet<Work>;

File: StockApi/Entities/Work.cs
namespace StockApi.Entities
{
    public class Work
    {
        public string Id { get; set; }
        public string ProjectName { get; set; }
        public Dictionary<string, int> Master { get; set; }
        // public Dictionary<string, int> Match { get; set; }
    }
}
